### DISCUSSION TITLE
Monthly ESMValTool community meeting Febrary 2023

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#73] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next monthly ESMValTool community meeting will be in February. The result of the poll is that the meeting will take place on

Monday, February 6 at 3 PM CET

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

**Format and agenda**

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in https://github.com/ESMValGroup/Community/discussions/66#discussioncomment-4420163

**Comment by malininae** [comment]
If there will be a chance, I'd like to hear the other's opinion on https://github.com/ESMValGroup/ESMValTool/issues/3017 and https://github.com/ESMValGroup/ESMValCore/issues/1895

**Reply by zklaus** [reply]
@malininae mentioned an issue with the ERA5 data, where the first 7 hours of the first day of 1950 are missing. We suggest first contacting ECMWF about this possible issue, then following through with @malininae's suggestion to mask missing data.

**Comment by malininae** [comment]
Also can discuss https://github.com/ESMValGroup/ESMValCore/issues/1915 . Sorry, I usually gather a bunch of stuff :-)

**Reply by zklaus** [reply]
For the rolling window issue, we should probably upgrade the Iris routine https://github.com/SciTools/iris/blob/7da248cc8c8b30902bf2343b4945da373f9c58ab/lib/iris/util.py#L272

to make use of https://docs.dask.org/en/latest/generated/dask.array.lib.stride_tricks.sliding_window_view.html.

**Comment by TomasTorsvik** [comment]
Mask data contaminates some ocean transect plots ESMValGroup/ESMValTool#2893

**Reply by zklaus** [reply]
It seems the issue may have been resolved in the development version. @ledm and @valeriupredoi will have a look to confirm.

**Comment by zklaus** [comment]
@glpotter mentioned two issues with ESMValTool output to be uploaded to ESGF. One is a missing `tracking_id`, the other a lack of a `record` or `unlimited` dimension for `time`.  We agreed that @glpotter will open an issue to collect all things that need to be improved to make the output ESGF ready.

**Reply by zklaus** [reply]
Cf ESMValGroup/ESMValTool#3038.
